tests/baselines/reference/user/acorn.log

```diff
@@ -20,6 +20,13 @@ node_modules/acorn/bin/_acorn.js(63,23): error TS2769: No overload matches this
 node_modules/acorn/bin/run_test262.js(3,21): error TS2307: Cannot find module 'test262-parser-runner'.
 node_modules/acorn/dist/acorn.es.js(36,1): error TS2322: Type 'null' is not assignable to type 'string'.
 node_modules/acorn/dist/acorn.es.js(36,32): error TS2322: Type 'null' is not assignable to type 'string'.
+node_modules/acorn/dist/acorn.es.js(150,8): error TS2554: Expected 2 arguments, but got 1.
```

Comment:
there are a lot of these, probably from #37173

Reply:
Maybe; but that change only updates `isOptionalParameter` to use the stronger arity `getMinArgumentCount` call, and that, in turn, is only used by API functions and node serialization. It would only be if somewhere we were looking at `signature.minArgumentCount` directly that it could change (which is only used by signature cloning, `isPotentiallyUncalledDecorator` (which should probably use a call to `getMinArgumentCount` tbh), and `getMinArgumentCount`, so that can't be the case).